packages/app/src/helpers/stringUtils.ts

```diff
@@ -26,3 +29,27 @@ export const createPatientName = (patient: any, lastNameFirst?: boolean): string
 export const createSlugUrl = (slug: string | undefined): string => {
   return `${import.meta.env.VITE_APP_BASE_URL}/${slug}`;
 };
+
+export const createProvider = (providerProfile: Practitioner | undefined): ProviderData => {
+  // TODO: fix the type of obj
```

Comment:
Just commenting on your comment, this looks like a nice spot for a generic input